### ISSUE TITLE
Propshaft setup raises `assets_manifest` undefined method

### ISSUE THREAD

**Issue by pedrofs** [avo-hq/avo#1218]
### Describe the bug
I'm setting up a Rails 7 + propshaft application, and after installing avo, I get the error below:
<img width="998" alt="image" src="https://user-images.githubusercontent.com/56484/188885059-7fb8754d-d9f9-4d57-a674-0ed9495cbb65.png">

The issue is on the `SvgFinder#default_strategy` method. The default strategy seems to have been designed to work with Sprockets. If I remove the default strategy entirely and rely on the predefined static directory search, it works.

### Steps to Reproduce
1. rails new test_app -a propshaft
2. rails generate avo:install
3. Go to localhost:3000/avo

### Expected behavior & Actual behavior
The avo starting page should open, but the above error raises.

### System configuration
**Avo version**: 2.14.2

**Rails version**:  7.0.3.1

**Ruby version**: 3.1

**License type**:

 - [x] Community
 - [ ] Pro

#### Impact

 - [x] High impact
 - [ ] Medium impact
 - [ ] Low impact

#### Urgency

 - [ ] High urgency
 - [ ] Medium urgency
 - [x] low urgency

**Comment by pedrofs** [comment]
Pointing to the main has fixed the issue.

**Comment by adrianthedev** [comment]
Yes, we made some tweaks to acomodate Propshaft. They are going to be released on Tuesday
